symbols/overlay10.yml

```diff
@@ -219,6 +219,29 @@ overlay10:
         
         r0: move_id
         return: anim_ent_ptr (This might be a mistake? It seems to be an integer, not a pointer)
+    - name: DrawTeamStats
+      address:
+        EU: 0x22C1328
+      description: "Handles creating the windows, sprites, etc. for the team stats top screen display."
+    - name: UpdateTeamStats
+      address:
+        EU: 0x22C1620
+      description: |-
+        Handles updating the team stats top screen display.
+        
+        No params.
+    - name: FreeTeamStats
+      address:
+        EU: 0x22C1CF4
+      description: |-
+        Handles the procedure to close the team stats top screen display.
+        
+        First it deletes the sprites, then it closes the portrait boxes and then the text
+        boxes containing the stats for all 4 team members.
```

Comment:
Missing return value

Split line, see above

Reply:
`overlay10.h` says this returns an int, yet you updated it to "no params" (which also implies no return). Which one is right?

Reply:
Fixed